GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+
+### Stage 3: conflict resolution
+If the proposed change cannot gather a 2/3 majority of the votes cast, the CC tries to resolve the deadlock. 
+
+* Any CC decision must be backed by an [enhancement proposal](#sktime-enhancement-proposals).
+* The CC will use consensus seeking.
+* If no consensus can be found within a month, the decision is made through a simple majority vote (with tie breaking) among the CC members. 
+
+### sktime enhancement proposals (STEPs)
+For all decision of the CC, an enhancement proposal must have been made public and discussed before the vote. 
+
+An enhancement proposal must be a consolidated document, with a concise problem statement, a clear description of the proposed solution and a comparison with alternative solutions, as outlined in our [template](https://github.com/sktime/enhancement-proposals/blob/master/TEMPLATE.md). 
+
+We collect and discuss proposals in sktime's [repository for enhancement-proposals](https://github.com/sktime/enhancement-proposals). 
+
+### Algorithm inclusion criteria
+
+Curation is about how we select contributions, which criteria we use in order to decide which contributions to include, and in which cases we deprecate and remove contributions.
+
+We have the following guidelines:
+
+* We only consider published algorithms which have been shown to be competitive in comparative benchmarking studies or practically useful in applied projects. A technique that provides a clear-cut improvement (e.g. an enhanced data structure or a more efficient approximation technique) on a widely-used method will also be considered for inclusion.
+* From the algorithms or techniques that meet the above criteria, only those which fit well within the current framework and API of sktime are accepted. For extending current frameworks and API, see the [process for major changes]().
+* The contributor should support the importance of the proposed addition with research papers and/or implementations in other similar packages, demonstrate its usefulness via common use-cases/applications and corroborate performance improvements, if any, with benchmarks and/or plots. It is expected that the proposed algorithm should outperform the methods that are already implemented in sktime in at least some areas.
+* We strive to consolidate existing functionality if helps to improve the usability and maintainability of the project. For example, when there are multiple techniques for the same purpose, we prefer to combine them into a single class and make case distinctions based on hyper-parameters.
+
+Note that your implementation need not be in sktime to be used together with sktime tools. You can implement your favorite algorithm in a sktime
+compatible way in one of [our companion repositories](https://github.com/sktime) on GitHub. We will be happy to list it under [related software](https://github.com/alan-turing-institute/sktime/wiki/related-software).
```

Comment:
perhaps explain that there are multiple options: companion/affiliated, or implement the interface in your package without necessarily becoming a "companion" or working in the sktime github org

Reply:
Yes, we need to think about the package affiliation and related governance process, but we can always add this later